lambdas/preview/index.py

```diff
@@ -33,15 +31,14 @@
 Draft4Validator.check_schema(SCHEMA)
 VALIDATOR = Draft4Validator(SCHEMA)
 
-
 def lambda_handler(event, _):
     """
     dynamically handle preview requests for bytes in S3
 
     caller must specify input_type (since there may be no file extension)
     """
-    params = event['queryStringParameters'] or {}
-    headers = event['headers'] or {}
+    params = event.get('queryStringParameters', {})
```

Comment:
Uhh that was intentional - that's how AWS gives you the data. If there are no query params, you get a `None`. It's stupid... But, need to adjust the unit test to simulate AWS, not adjust the code to fit the unit test :-P

Reply:
Oh, OK, I'll revert that